tools/cjxl.cc

```diff
@@ -617,7 +619,7 @@ jxl::Status CompressArgs::ValidateArgs(const CommandLineParser& cmdline) {
 
   if (got_distance) {
     constexpr float butteraugli_min_dist = 0.1f;
-    constexpr float butteraugli_max_dist = 15.0f;
+    constexpr float butteraugli_max_dist = 20.0f;
```

Comment:
Maybe bump it up a bit further? Otherwise a new encode technique is introduced for only one very specific distance target (20).

Also: maybe reconsider also updating the -q setting at which we switch to modular?

Reply:
I don't mind extending it to 25, done

about the -q setting: that's an independent discussion, perhaps better done e.g. in a bug report than in this PR